pkg/devfile/header.go

```diff
@@ -43,6 +43,10 @@ type DevfileMetadata struct {
 	// +optional
 	Tags []string `json:"tags,omitempty"`
 
+	// Optional devfile architecture, amd64 is considered the default architecture unless specified otherwise
```

Comment:
"Optional devfile architecture" sounds a little unclear to me.

Maybe something like "Optional list of processor architectures that this devfile supports"?